src/pages/library/RequestDatasetAccessModal.js

```diff
@@ -71,22 +72,26 @@ export const RequestDatasetAccessModal = ({ onDismiss, datasets }) => {
   ])
 }
 
-const RequestDatasetAccessButton = ({ dataset }) => {
+const RequestDatasetAccessButton = (dataset) => {
```

Comment:
Being a react component `dataset` is essentially the list of props being passed in to this component. We destructure the props on the component. Also, using Christine's suggestion above, this could be changed to `({title, id}) => {`
I would prefer this approach in the spirit of using the least powerful abstraction / simplicity. Given the amount of data in the dataset, I would prefer only passing in the necessary information for a small component like this.